STANDARDS.md

```diff
@@ -1,16 +1,66 @@
-# Dataset Standards
 
-This file outlines the standards and requirements needed for submitting a dataset to QCArchive.
+This file outlines the standards and data requirements needed for submitting a dataset to QCArchive.
 This ensures that we have a consistent data model for downstream processes.
 
-## Required fields 
+STANDARDS version: 3-DRAFT (2020-11-13)
+
+We distinguish between standards for the datasets (i.e. the actual data), and the standards for training force fields.
 
-Current list:
-* Ensure all submissions have cmiles, most important are mapped hydrogen smiles
-* Ensure the WBO is requested for all submissions, this should be included in the scf properties list using the flag `wiberg_lowdin_indices`
+# Dataset Standards
 
-## Best practices
-* If any calculations are to be redone from another collection, re-use the old input (coordinates, atom ordering etc) as this will avoid running the calculation again and will just create new references in the database to the old results and should help keep the cost of the calculations down.  
+## Each molecule must have the following information:
+- Canonical isomeric explict hydrogen mapped SMILES
+- Provenence info of SMILES generation (NEW)
+- Coordinates
+- Total Charge
+- Coordinates must be in CMILES order (discuss)
+
+## Each dataset must have the following information:
+- Name
+- Version (NEW)
+- A short description
+- A long description
+- A link/URL/reference pointing to the provenence to reproduce dataset (e.g. the GH submission folder)
+- A changelog (NEW)
+- A github submitter username
+- The name of the person who selected/sourced the molecules (NEW)
+- A description of the meaning of the entry/molecule keys/names (NEW)
+- Each entry has canonical isomeric explict hydrogen mapped SMILES
+- Provenence info of CMILES generation (NEW)
+- A set of elements that the dataset contains
+- A set of charges that the dataset contains (NEW)
+- The mean and max molecular weight of molecules the dataset contains (NEW)
```

Comment:
Again this will be great!

Reply:
Should we instead use heavy atoms as a count? I am trying to encapsulate the size of molecules that the dataset contains in a simple way

Reply:
Molecule weight involves isotopes, and as far as I know these shouldn't affect most of our calculations. I agree that we could do both for the sake of doing both. On the QM side, I guess number of electrons is more useful, but on the MM/informatics side, I am thinking weight could be a better descriptor here.